_extensions/onlimit-theme/_extension.yml

```
@@ -0,0 +1,35 @@
+title: onlimit-theme
+author: |
+  - Signe Kirk Brødbæk
+  - Daniel B. Ibsen
+  - Luke W. Johnston
+version: 1.0.0
+quarto-required: ">=1.6.0"
+contributes:
+  formats:
+    html:
+      toc: true
+      from: markdown+emoji
+
+  project:
+    project:
+      type: website
+
+    website:
+      favicon: favicon/favicon.ico
+      page-navigation: true
+      navbar:
+        pinned: true
+        tools:
+        - icon: github
+          href: "https://github.com/onlimit-study/"
+          aria-label: "GitHub icon: Link to ON-LiMiT organization page"
+      search:
+        location: navbar
+        type: textbox
+      page-footer:
+        center:
+          - text: "License: CC BY 4.0"
+            href: https://creativecommons.org/licenses/by/4.0/
+
+    brand: _brand.yml
```

```suggestion
    brand: _brand.yml

```

We'll eventually add pre-commit to fix these end of file issues.